presto-iceberg/src/test/java/com/facebook/presto/iceberg/procedure/TestSetTablePropertyProcedure.java

```diff
@@ -160,10 +166,22 @@ public void testInvalidSetTablePropertyProcedureCases()
 
     private Table loadTable(String tableName)
     {
+        tableName = normalizeIdentifier(tableName);
         Catalog catalog = CatalogUtil.loadCatalog(HadoopCatalog.class.getName(), ICEBERG_CATALOG, getProperties(), new Configuration());
         return catalog.loadTable(TableIdentifier.of(TEST_SCHEMA, tableName));
     }
 
+    protected String normalizeIdentifier(String name)
```

Comment:
May be move this function to `AbstractTestQueryFramework`

Reply:
Ohh yes, I missed this in the cleanup. Thanks, will update